lib/scraping/hpi_web_scraper.rb

```diff
@@ -0,0 +1,45 @@
+require "#{Rails.root}/lib/scraping/scraping_exception.rb"
+
+# "Abstract" class
+class HpiWebScraper
+  @@phone_words = %w[Tel.: Telefon: telephone: Telephone: phone: Phone: Phone.: Tel:]
+  @@office_words = %w[office: Office: Raum: Room:]
+  @@email_words = %w[Email: E-mail: E-Mail: e-mail:]
+
+  @@base_url = 'https://hpi.de'
+
+  def initialize(html_document)
+    @html = html_document
+  end
+
+  def scrape
+    raise "Function has to be overridden."
+  end
+
+  def download_image(person_image_div)
+    image = person_image_div.at_css('img')
+    unless image
+      # No image on website
+      return
+    end
+
+    img_src = image.attr('src')
+    img_src = @@base_url + img_src
+
+    get_image_file_path_and_save(img_src)
+  end
+
+  def get_image_file_path_and_save(img_src)
+    # Get the file name without complete file path
+    index = img_src.rindex("/")
+    file_name = img_src[index + 1, img_src.length - 1]
+
+    relative_file_path = "/app/assets/images/people/#{file_name}"
+    file_path = "#{Rails.root}#{relative_file_path}"
```

Comment:
There's a nice join function.

```suggestion
    relative_file_path = "app/assets/images/people/#{file_name}"
    file_path = Rails.root.join(relative_file_path)
```